apps/microsoft-teams/app-actions/src/constants.ts

```diff
@@ -5,4 +5,5 @@ export const TOPIC_ACTION_MAP = {
   'ContentManagement.Entry.publish': 'published',
   'ContentManagement.Entry.unpublish': 'unpublished',
   'ContentManagement.Entry.delete': 'deleted',
+  'Workflow.Step.notifyMicrosoftTeams': 'Workflow.Step.notifyMicrosoftTeams',
```

Comment:
Right I see your copy. Let's just remove this line and check the topic string against the literal value as I commented above.

Reply:
We could just make it an official cosntant though (separate from TOPIC_ACTION_MAP)

```ts
export const WORKFLOW_STEP_EVENT = 'Workflow.Step.notifyMicrosoftTeams'
```

Reply:
Yeah, that seems like a reasonable step in the right direction if we need an official const/enum/type elsewhere in the codebase.  Let me take a look.

- [ ] investigate applicabilty of `export const WORKFLOW_STEP_EVENT = 'Workflow.Step.notifyMicrosoftTeams'`
- [ ] implement if needed.